utils/urlEncoder.js

```diff
@@ -0,0 +1,17 @@
+require("dotenv").config();
+
+const constructEncodedSALReferralUrl = (
+  name,
+  email,
+  campaign = "Edustipend at 2",
+  source
+) => {
+  const referralUrl = `https://www.edustipend.org/support-a-learner?utm_campaign=${campaign}&utm_medium=${email}&utm_referrer=${name}&utm_source=${
+    source ?? ""
```

Comment:
Just out of curiosity, what are the different sources we can have? (Referral link systems are new to me)

Reply:
Here: https://buffer.com/library/utm-guide/